service/src/test/java/uk/gov/hmcts/reform/fpl/service/FurtherEvidenceNotificationServiceTest.java

```diff
@@ -225,6 +273,14 @@ private CaseData caseData(Representative... representatives) {
             .build();
     }
 
+    private CaseData caseData(Child children, Respondent respondents) {
```

Comment:
what is the point of passing a child and respondent here?

Reply:
not sure that the method is even used

Reply:
Thank you for the review. It's because I have written a test which needs casedata to have children and respondent fields :)